.gitignore

```diff
@@ -1,15 +1,11 @@
 *.iml
 .gradle
 /local.properties
-/.idea/caches
```

Comment:
The issue was, there were only some files excluded, not all. I changed this, so the whole folder is excluded now. I deleted it from this pull.